terraform/modules/lambda/main.tf

```diff
@@ -47,7 +47,7 @@ resource "aws_lambda_function" "function" {
   description   = var.description
   handler       = var.handler
   role          = aws_iam_role.role[0].arn
-  runtime       = "python3.6"
+  runtime       = "python3.7"
```

Comment:
AWS Lambda has [Python3.8](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html) available, should we use that?